src/Main/Java/com/hillel/vynnyk/homeworks/homework7/FileNavigator.java

```diff
@@ -0,0 +1,89 @@
+package com.hillel.vynnyk.homeworks.homework7;
+
+import java.util.*;
+
+
+public class FileNavigator {
+    Map<String, ArrayList<FileData>> map = new HashMap<>();
+
+    public Map<String, ArrayList<FileData>> add(String path, FileData file) {
+        ArrayList<FileData> data = new ArrayList<>();
+        if (map.containsKey(path)) {
+            map.get(path).add(file);
+        } else {
+            data.add(file);
+            map.put(path, data);
+        }
+        System.out.println(map);
+        return map;
+    }
+
+    public ArrayList<FileData> find(String key) {
+        if (map.containsKey(key)) {
```

Comment:
```suggestion
       return map.get(key);
```
воно само дасть null